internal/handlers/pg/msg_hostinfo.go

```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 	"time"
 
+	"gopkg.in/ini.v1"
```

Comment:
Let's not use an additionaly dependency for something simple as this task

Reply:
Fixed